iidm-network-api/src/main/java/eu/itesla_project/iidm/network/EquipmentTopologyVisitor.java

```diff
@@ -12,7 +12,7 @@
  */
 public abstract class EquipmentTopologyVisitor extends AbstractTopologyVisitor {
 
-    public abstract void visitEquipment(Connectable eq);
+    public abstract <I extends Connectable<I>> void visitEquipment(Connectable<I> eq);
```

Comment:
That was the solution I was using locally. For the proposed change I tried to preserve the more strict definition of connectable, that continues working for my code. For my visitors I only need to rely on the fact that Connectable extends the identifiable interface.